### DISCUSSION TITLE
【教程】屏蔽B站“粉丝”选项

### DISCUSSION THREAD

**Discussion by win2045** [share121/inter-knot#5047] [教程]
# 因本人经常会逛B站，但经常会看到例如原神官号粉丝经常在掉而影响到心情，于是决定*掩耳盗铃*   ~让自己有个好心情~

## 废话少说，进入正题
1. 首先在应用商店下载这个，是其他浏览器请选择自带扩展商店
![image](https://github.com/user-attachments/assets/e5a4af0e-2644-419f-8980-c4d4a8f5691a)

2. 下载之后点击扩展程序你会看到一个这样的界面
![image](https://github.com/user-attachments/assets/d3c9433f-47b3-47c5-b2be-2ab59c5ace38)

3. 右键想要屏蔽的区域后选择
![image](https://github.com/user-attachments/assets/f01a93d3-10e6-4ae9-b146-f1226d779048)
然后出现一个红框，右下角选择创建便直接屏蔽框定区域

4. 过滤规则你可以选择不同的区域
![image](https://github.com/user-attachments/assets/655a6de8-0f6d-40c3-a15c-bd5977a8f8e7)
![image](https://github.com/user-attachments/assets/4c06bf7e-5bb4-4dcb-8af3-e9aceafb751b)

5. 如何删除刚才屏蔽的选项
![image](https://github.com/user-attachments/assets/90e365aa-a39d-4b07-a40d-003e036e32d9)
![image](https://github.com/user-attachments/assets/35f376ed-6274-440d-a834-bdbdfae1cbe5)
![image](https://github.com/user-attachments/assets/b401c85f-e4c3-42ad-8a45-28b542f6c300)直接把这行多出来的删掉
然后左上角选择应用修改，重新刷新一下就回来了

## 最后是效果图
![image](https://github.com/user-attachments/assets/93b7880a-e135-46df-a173-8c176b953275)
![image](https://github.com/user-attachments/assets/2a5d0f6d-7ee0-43b4-9d95-62add6f8be2b)
![image](https://github.com/user-attachments/assets/5aff95a9-69d6-4ae9-aee9-db87ce5c10ca)

想要屏蔽点击头像显示出来的粉丝也可以直接把指针点着想要屏蔽的位置后右键用小键盘的*上下*来选择屏蔽元素后Enter然后创建

最后希望大家都不会被节奏影响到游玩时的心情

**Comment by Minapocalis** [comment]
原神粉丝掉其实也没多大影响，只需要知道一件事情掉的都是黑粉，这是他们一直常用的方法先关注等有节奏的之后再取关，

**Reply by Minapocalis** [reply]
在B站粉丝列表你点开你就能看见小黑子

**Reply by win2045** [reply]
是这样，像牛皮藓一样死皮赖脸

**Comment by yuehe27** [comment]
其实，我觉着人少挺好的。小圈子的美好你真的想象不到。
